### PR TITLE
Awesome Books : refactor to use Javascript classes

### PR DESCRIPTION
**Pull request by shihabrafio**
In this project, We were able to

- Use JavaScript classes instead of objects and arrays.
- Create class methods to add and remove books.
- Maintain all of the original functionality of the application.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @shihabrafio  @Yordinia, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters, kindly fix this

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+h1 {
+  font-size: 40px;
+}
+
+header h1 {
+  margin-top: 4%;
+}
+
```
Comment (thecodechaser) [inline style.css:17]: - [ ] You did a good job on the functionalities but you need to fix the below design issues.
  - Please make sure to replace the heading `Awsome Books` with `All awesome books`.
  - Please make sure the remove book button is aligned at the end of the block even if the book title and author names are very less in length.

### Your design

image

### Wireframe design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @shihabrafio, and @Yordinia👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- GitHub flow is followed. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Yordinia & @shihabrafio, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
- Has class methods to add and remove books ✔️
- refactoring project to use JavaScript classes ✔️
- Implement changes requested by the reviewer ✔️
- Good use of Github flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**